app/lib/actions/scc_manager/subscribe_product.rb

```diff
@@ -35,7 +35,6 @@ def finalize
         scc_product = SccProduct.find(input[:scc_product][:id])
         product = ::Katello::Product.find(input[:product_id])
         scc_product.update!(product: product)
-        ::Foreman::Logging.logger('foreman_scc_manager').warn input[:scc_product]
```

Comment:
Can you remove this in the commit it was introduced with?

Reply:
Fixed.